woodwork/type_sys/inference_functions.py

```diff
@@ -110,6 +110,12 @@ def boolean_nullable_func(series):
                 return True
         except TypeError:  # Necessary to check for non-hashable values because of object dtype consideration
             return False
+    elif pdtypes.is_integer_dtype(series.dtype) and len(
+        config.get_option("boolean_inference_ints"),
+    ):
+        series_unique = set(series)
+        if series_unique == set(config.get_option("boolean_inference_ints")):
+            return True
```

Comment:
This only works for a value set as a list like `[0, 1]` right? We should consider making this capable of handling a list of lists so that the format matches that of `boolean_inference_strings`.
```
elif pdtypes.is_integer_dtype(series.dtype) and len(
        config.get_option("boolean_inference_ints"),
    ):
    series_unique = set(series)
    for boolean_list in config.get_option("boolean_inference_ints"):
        if series_unique == set(boolean_list):
            return True
```
What do you think?

Reply:
Hmm, I don't think we should look towards automating other possible `int` inputs. Imo, there shouldn't be other int values that can be considered boolean the way `[0, 1]` are. What other possible values were you thinking?

Reply:
I agree that I can't personally think of other cases where open source users might want to add to this list, but if they did try they'd be limited in this regard. Either way I think it's totally fine if we don't add this here, it's not a priority functionality

Reply:
I think it's fine to leave this out!